fallanic/GifLens
# Adding languages

Ticket: https://github.com/fallanic/GifLens/projects/1#card-19924294 
https://github.com/fallanic/GifLens/projects/1#card-19932020 <br>

## Changes

- Added all the languages I could using that list https://code.visualstudio.com/docs/languages/identifiers and also searching in the codebase that the scopeNames are accurate for each of them
![image](https://user-images.githubusercontent.com/1240520/56505229-b09c5c80-64e0-11e9-981b-6d84a2130445.png)
- updated the config in `package.json` and `search.ts`
- we are handling single line comments, and in some case comments with a start + end syntax

## How To Test

- run the extension, create files for different languages and try adding GITLENS tags, notice they are using syntax coloring

## Applicable Research Resources

- https://code.visualstudio.com/docs/languages/identifiers
